stdlib/2/inspect.pyi

```diff
@@ -55,11 +55,11 @@ def ismemberdescriptor(object: object) -> bool: ...
 def findsource(object: object) -> Tuple[List[str], int]: ...
 def getabsfile(object: object) -> str: ...
 def getblock(lines: Sequence[str]) -> Sequence[str]: ...
-def getdoc(object: object) -> str: ...
-def getcomments(object: object) -> str: ...
+def getdoc(object: Optional[object]) -> Optional[str]: ...
+def getcomments(object: Optional[object]) -> Optional[str]: ...
 def getfile(object: object) -> str: ...
-def getmodule(object: object) -> ModuleType: ...
-def getsourcefile(object: object) -> str: ...
+def getmodule(object: Optional[object]) -> Optional[ModuleType]: ...
```

Comment:
Using `Optional` for the arguments in redundant. After all, `None` is an object as well.

Reply:
Ah! Dropped that commit and force-pushed just the other. Another PR upcoming for `Optional[object]` elsewhere in the tree.